apps/meteor/app/integrations/server/triggers.ts

```diff
@@ -8,7 +8,12 @@ const callbackHandler = function _callbackHandler(eventType: string) {
 	};
 };
 
-callbacks.add('afterSaveMessage', callbackHandler('sendMessage'), callbacks.priority.LOW, 'integrations-sendMessage');
+callbacks.add(
+	'afterSaveMessage',
+	(message, { room }) => callbackHandler('sendMessage')(message, room),
```

Comment:
I'm not sure if this will fix the issue, since we changed the parameters of the callback here: https://github.com/RocketChat/Rocket.Chat/pull/33034/files#diff-deed993f8ad9d2963a4a2b6a22c0d629078a2a4895f1fcf7507d2c94901c56c7L53

The `sendMessage` handler is expecting 3 parameters and since we removed/replaced uid and roomUpdater, I'm not sure if it really fixes it.
Validation here: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/integrations/server/lib/triggerHandler.js#L131